fe/fe-core/src/main/java/com/starrocks/lake/StarOSAgent.java

```diff
@@ -472,4 +472,15 @@ public void updateMetaGroup(long metaGroupId, List<Long> shardGroupIds, boolean
             throw new DdlException("Failed to update meta group. error: " + e.getMessage());
         }
     }
+
+    // Mocked
+    public long createWorkerGroup(String size) throws DdlException {
+        return GlobalStateMgr.getCurrentState().getNextId();
```

Comment:
can we actually create warehouse in this pr?

Reply:
no, because starMgr with worker group crud has not been published yet, so I have no choice but to mock it for merging.

Reply:
ok